charts/devlake/values.yaml

```diff
@@ -121,7 +121,6 @@ mysql:
 
 #   containerSecurityContext: {}
 
-externalGrafana:
```

Comment:
why deleting this line?

Reply:
@klesh good catch, thanks. This should be fixed now